client/components/DateFormat.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
Still figuring out prop-types myself but I believe it would look something like this is you're looking to get the eslint errors to go away

```
DateFormat.propTypes = {
  rawDate: PropTypes.instanceOf(Date)
};
```

Reply:
Ill give it a try thanks.